modules/jaeger-deploy-production-streaming.adoc

```diff
@@ -0,0 +1,140 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-deploying.adoc
+////
+
+[id="jaeger-deploy-streaming_{context}"]
+= Deploying the Jaeger streaming strategy
+
+The `streaming` deployment strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required.
+
+The `streaming` strategy provides a streaming capability that sits between the collector and the storage (Elasticsearch). This reduces the pressure on the storage under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (Kafka).
+
+.Prerequisites
+
+* The Elasticsearch Operator must be installed.
+* An Elasticsearch instance must be deployed.
+* The Jaeger Operator must be installed.
+* Review the instructions for how to customize the Jaeger installation.
+* An account with the `cluster-admin` role.
+
```

Comment:
Also requires `AMQ Streams Operator` - if using version 1.4.0 or higher, then can use Kafka self provisioning, otherwise user would need to create the Kafka instance.